### PR TITLE
Component: Card

### PR DESCRIPTION
**Pull request by haild173094**

### CODE REVIEW

File: src/components/Card/Card.vue [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  background: 'bg-surface',
+  padding: '400',
+  roundedAbove: 'sm',
+});
+
+const breakpoints = useBreakpoints();
+const defaultBorderRadius: BorderRadiusAliasOrScale = '300';
+const hasBorderRadius = computed<boolean>(() => !!breakpoints.value[`${props.roundedAbove}Up`]);
+
+watch(
```
Comment (juzser) [inline src/components/Card/Card.vue:45]: .

File: src/components/Card/types.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  BreakpointsAlias,
+  ColorBackgroundAlias,
+  SpaceScale,
+} from '@shopify/polaris-tokens';
+import type {ResponsiveProp} from '@/utilities/css';
+
+type Spacing = ResponsiveProp<SpaceScale>;
+
+export interface CardProps {
+  background?: ColorBackgroundAlias;
```
Comment (juzser) [inline src/components/Card/types.ts:11]: Is it missing comment?

**Review by juzser — CHANGES_REQUESTED** [review]

File: src/components/ShadowBevel/types.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import type {
+  BorderRadiusAliasOrScale,
+  ShadowAliasOrScale,
+} from '@shopify/polaris-tokens';
+
+import type {ResponsiveProp} from '@/utilities/css';
```
Comment (HQCuong) [inline src/components/ShadowBevel/types.ts:6]: missing space

File: src/components/Card/Card.vue [hunk 4]
```diff
@@ -0,0 +1,44 @@
+<template lang="pug">
+ShadowBevel(
+  :box-shadow="'100'",
```
Comment (HQCuong) [inline src/components/Card/Card.vue:3]: box-shadow="100"

File: src/components/ShadowBevel/ShadowBevel.vue [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  :is="as",
+  :className="styles.ShadowBevel",
+  :style="style",
+)
+  slot
+</template>
+
+<script setup lang="ts">
+import { computed, useCssModule } from 'vue';
+import {getResponsiveValue} from '@/utilities/css';
```
Comment (HQCuong) [inline src/components/ShadowBevel/ShadowBevel.vue:12]: missing space

File: src/components/ShadowBevel/ShadowBevel.vue [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+import type {ResponsiveProp} from '@/utilities/css';
```
Comment (HQCuong) [inline src/components/ShadowBevel/ShadowBevel.vue:13]: .

File: src/components/Card/Card.vue [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+    overflow-x="hidden",
+    overflow-y="hidden",
+    min-height="100%",
+  )
+    slot
+</template>
+
+<script setup lang="ts">
+import { computed } from 'vue';
+import type {
```
Comment (HQCuong) [inline src/components/Card/Card.vue:19]: optional: don't need to break line here

File: src/components/Card/Card.vue [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+defineSlots<CardSlots>();
+
+const props = withDefaults(defineProps<CardProps>(), {
+  background: 'bg-surface',
+  padding: '400',
+  roundedAbove: 'sm',
+});
+
+const breakpoints = useBreakpoints();
+const defaultBorderRadius: BorderRadiusAliasOrScale = '300';
```
Comment (HQCuong) [inline src/components/Card/Card.vue:42]: breakline

**Review by HQCuong — CHANGES_REQUESTED** [review]
convention

File: docs/components/Card/AllElement.vue [hunk 9]
```diff
@@ -0,0 +1,123 @@
+<template>
+
```
Comment (juzser) [inline docs/components/Card/AllElement.vue:2]: .

File: docs/components/Card/SectionAndAction.vue [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+      </Text>
+    </BlockStack>
+    <BlockStack gap="200">
+      <InlineGrid columns="1fr auto">
+        <Text as="h3" variant="headingSm" fontWeight="medium">
+          Contact Information
+        </Text>
+        <Button
+          variant="plain"
+          onClick={() => {}}
```
Comment (juzser) [inline docs/components/Card/SectionAndAction.vue:19]: .

File: docs/components/Card/SectionAndCriticalAction.vue [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+      <InlineGrid columns="1fr auto">
+        <Text as="h3" variant="headingSm" fontWeight="medium">
+          Contact Information
+        </Text>
+        <ButtonGroup>
+          <Button
+            variant="plain"
+            tone="critical"
+            accessibilityLabel="Delete"
+            @Click="() => {}"
```
Comment (juzser) [inline docs/components/Card/SectionAndCriticalAction.vue:22]: .

File: docs/components/Card/SectionAndCriticalAction.vue [hunk 12]
```diff
@@ -0,0 +1,44 @@
...
+          >
+            Delete
+          </Button>
+          <Button
+            variant="plain"
+            accessibilityLabel="Edit"
+            @Click="() => {}"
```
Comment (juzser) [inline docs/components/Card/SectionAndCriticalAction.vue:29]: .

File: docs/components/Card/WithMultipleFooterAction.vue [hunk 13]
```diff
@@ -0,0 +1,55 @@
...
+    </BlockStack>
+    <InlineStack align="end">
+      <ButtonGroup>
+        <Popover
+          active={actionActive}
+          onClose={handleToggleAction}
+        >
+          <ActionList :items="items" />
+          <template #activator>
+            <Button disclosure accessibilityLabel="More" @Click="handleToggleAction">
```
Comment (juzser) [inline docs/components/Card/WithMultipleFooterAction.vue:24]: . Click

File: docs/components/Card/WithSeparateHeader.vue [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+      <ButtonGroup>
+        <Button
+          variant="plain"
+          accessibilityLabel="Preview"
+          @click="() => {}"
+        >
+          Preview
+        </Button>
+        <Popover
+          active={actionActive}
```
Comment (juzser) [inline docs/components/Card/WithSeparateHeader.vue:17]: .

File: docs/components/Card/WithSeparateHeader.vue [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+          activator={disclosureButtonActivator}
```
Comment (juzser) [inline docs/components/Card/WithSeparateHeader.vue:18]: .

**Review by juzser — CHANGES_REQUESTED** [review]

File: docs/components/Card/SectionAndAction.vue (hunk 10 above)
Comment (juzser) [reply docs/components/Card/SectionAndAction.vue:19]: `@click="..."`

**Review by juzser — APPROVED** [review]

**Pull request merged**